drivers/hmis/lib/form_data/assessments/dummy_intake_assessment.json

```diff
@@ -41,44 +30,148 @@
         {
           "type": "CHOICE",
           "required": false,
-          "link_id": "3.917",
+          "link_id": "3.917-1",
           "text": "Prior Living Situation",
-          "pick_list_reference": "LivingSituation"
+          "pick_list_reference": "PRIOR_LIVING_SITUATION"
         },
         {
           "type": "CHOICE",
           "link_id": "3.917-2",
-          "text": "Length of stay in prior living situation"
+          "text": "Length of stay in prior living situation",
+          "pick_list_reference": "LengthOfStay"
         },
         {
-          "type": "BOOLEAN",
-          "link_id": "3.917-A",
-          "text": "Did the client stay less than 90 days?"
-        },
-        {
-          "type": "BOOLEAN",
-          "link_id": "3.917-B",
-          "text": "Did the client stay less than 7 nights?"
-        },
-        {
-          "type": "BOOLEAN",
-          "link_id": "3.917-C",
-          "text": "On the night before, did the client stay on the streets, ES or SH?"
+          "type": "GROUP",
+          "link_id": "pls-conditionals",
+          "item": [
+            {
+              "type": "BOOLEAN",
+              "link_id": "3.917-A",
+              "text": "Did the client stay less than 90 days?",
```

Comment:
I'm not sure if the form needs to explicitly ask this, since it can be derived from the Length of Stay answer

Reply:
I agree, this should be auto-calculated from the other responses.  Also 3.917-A and 3.917-2B use the same field.  (Same thing for the other one about auto calculation.)

Reply:
Okay gotcha. In that case, I think `3.917-A` and `3.917-B` should be removed from the form, and subsequent form logic that depends on their answers will be changed to look at the answer to `3.917-2` instead. (Also, I'll change all the `3.917-2`->`3.917.2` so it matches HUD, I didn't know the convention when I wrote this originally)